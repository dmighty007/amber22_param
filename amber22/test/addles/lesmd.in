leucine LES test case
 &cntrl

  ntx    = 5, ntwx   = 10,   nsnb   = 10,                dielc = 1.0,
              nstlim = 1000, ntpr   = 10, temp0  = 298., dt    = 0.001,
  ntt    = 1,                ntc    = 1,  tol    = 0.0005,
  scee   = 1.2,
  
 /
