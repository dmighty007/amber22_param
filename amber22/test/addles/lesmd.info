
 NSTEP =  1000  TIME(PS) =    1.000  TEMP(K) =   269.79  PRESS =      0.00
 Etot   =     212.5796  EKtot   =     126.2594  EPtot      =      86.3202
 BOND   =      31.6872  ANGLE   =      69.9273  DIHED      =      31.7964
 1-4 NB =       9.4088  1-4 EEL =      19.0791  VDWAALS    =       1.6832
 EELEC  =     -77.2618  EHBOND  =       0.0000  CONSTRAINT =       0.0000
