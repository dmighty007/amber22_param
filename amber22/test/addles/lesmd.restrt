for leu -> glu                                                                  
  157  0.1000000E+01
   6.9091002  -8.4666566   0.7509820   6.5428329  -7.5145611   0.9891422
   5.7838900  -7.4117268   1.6772189   6.2637290  -6.7529725   0.2305985
   7.7496925  -6.7396741   1.6584204   7.7656679  -6.3950971   2.7966304
   8.8354975  -7.1806564   0.9035568   8.8354975  -7.1806564   0.9035568
   8.8354975  -7.1806564   0.9035568   8.8354975  -7.1806564   0.9035568
   8.8354975  -7.1806564   0.9035568   8.4475737  -7.5446147   0.0235987
   8.4475737  -7.5446147   0.0235987   8.4475737  -7.5446147   0.0235987
   8.4475737  -7.5446147   0.0235987   8.4475737  -7.5446147   0.0235987
  10.2454613  -6.7937533   0.9874632  10.2454613  -6.7937533   0.9874632
  10.2454613  -6.7937533   0.9874632  10.2454613  -6.7937533   0.9874632
  10.2454613  -6.7937533   0.9874632  10.2307220  -6.1565798   2.0130261
  10.2307220  -6.1565798   2.0130261  10.2307220  -6.1565798   2.0130261
  10.2307220  -6.1565798   2.0130261  10.2307220  -6.1565798   2.0130261
  11.0928807  -7.9333864   1.5504957  11.0928807  -7.9333864   1.5504957
  11.0928807  -7.9333864   1.5504957  11.0928807  -7.9333864   1.5504957
  11.0928807  -7.9333864   1.5504957  10.3897214  -8.4044548   2.1133431
  10.3897214  -8.4044548   2.1133431  10.3897214  -8.4044548   2.1133431
  10.3897214  -8.4044548   2.1133431  10.3897214  -8.4044548   2.1133431
  11.3904264  -8.5541628   0.7917770  11.3904264  -8.5541628   0.7917770
  11.3904264  -8.5541628   0.7917770  11.3904264  -8.5541628   0.7917770
  11.3904264  -8.5541628   0.7917770  12.4050793  -7.6260982   2.2039047
  12.4050793  -7.6260982   2.2039047  12.4050793  -7.6260982   2.2039047
  12.4050793  -7.6260982   2.2039047  12.4050793  -7.6260982   2.2039047
  12.4050793  -7.6260982   2.2039047  12.4050793  -7.6260982   2.2039047
  12.4050793  -7.6260982   2.2039047  12.4050793  -7.6260982   2.2039047
  12.4050793  -7.6260982   2.2039047  12.3536518  -6.5332310   2.1730892
  12.3536518  -6.5332310   2.1730892  12.3536518  -6.5332310   2.1730892
  12.3536518  -6.5332310   2.1730892  12.3536518  -6.5332310   2.1730892
  12.3536518  -6.5332310   2.1730892  12.3536518  -6.5332310   2.1730892
  12.3536518  -6.5332310   2.1730892  12.3536518  -6.5332310   2.1730892
  12.3536518  -6.5332310   2.1730892  12.1818923  -7.7044741   3.5331441
  12.1818923  -7.7044741   3.5331441  12.1818923  -7.7044741   3.5331441
  12.1818923  -7.7044741   3.5331441  12.1818923  -7.7044741   3.5331441
  12.1818923  -7.7044741   3.5331441  12.1818923  -7.7044741   3.5331441
  12.1818923  -7.7044741   3.5331441  12.1818923  -7.7044741   3.5331441
  12.1818923  -7.7044741   3.5331441  13.1175581  -7.0871249   3.8815859
  13.1175581  -7.0871249   3.8815859  13.1175581  -7.0871249   3.8815859
  13.1175581  -7.0871249   3.8815859  13.1175581  -7.0871249   3.8815859
  13.1175581  -7.0871249   3.8815859  13.1175581  -7.0871249   3.8815859
  13.1175581  -7.0871249   3.8815859  13.1175581  -7.0871249   3.8815859
  13.1175581  -7.0871249   3.8815859  11.4285100  -7.2525082   3.9756145
  11.4285100  -7.2525082   3.9756145  11.4285100  -7.2525082   3.9756145
  11.4285100  -7.2525082   3.9756145  11.4285100  -7.2525082   3.9756145
  11.4285100  -7.2525082   3.9756145  11.4285100  -7.2525082   3.9756145
  11.4285100  -7.2525082   3.9756145  11.4285100  -7.2525082   3.9756145
  11.4285100  -7.2525082   3.9756145  12.7073744  -8.4208523   4.0562422
  12.7073744  -8.4208523   4.0562422  12.7073744  -8.4208523   4.0562422
  12.7073744  -8.4208523   4.0562422  12.7073744  -8.4208523   4.0562422
  12.7073744  -8.4208523   4.0562422  12.7073744  -8.4208523   4.0562422
  12.7073744  -8.4208523   4.0562422  12.7073744  -8.4208523   4.0562422
  12.7073744  -8.4208523   4.0562422  13.6608667  -8.0976781   1.4605418
  13.6608667  -8.0976781   1.4605418  13.6608667  -8.0976781   1.4605418
  13.6608667  -8.0976781   1.4605418  13.6608667  -8.0976781   1.4605418
  13.6608667  -8.0976781   1.4605418  13.6608667  -8.0976781   1.4605418
  13.6608667  -8.0976781   1.4605418  13.6608667  -8.0976781   1.4605418
  13.6608667  -8.0976781   1.4605418  14.4930073  -8.7717059   1.8736677
  14.4930073  -8.7717059   1.8736677  14.4930073  -8.7717059   1.8736677
  14.4930073  -8.7717059   1.8736677  14.4930073  -8.7717059   1.8736677
  14.4930073  -8.7717059   1.8736677  14.4930073  -8.7717059   1.8736677
  14.4930073  -8.7717059   1.8736677  14.4930073  -8.7717059   1.8736677
  14.4930073  -8.7717059   1.8736677  13.3157451  -8.8497403   0.7773289
  13.3157451  -8.8497403   0.7773289  13.3157451  -8.8497403   0.7773289
  13.3157451  -8.8497403   0.7773289  13.3157451  -8.8497403   0.7773289
  13.3157451  -8.8497403   0.7773289  13.3157451  -8.8497403   0.7773289
  13.3157451  -8.8497403   0.7773289  13.3157451  -8.8497403   0.7773289
  13.3157451  -8.8497403   0.7773289  14.2335565  -7.4455829   0.9340278
  14.2335565  -7.4455829   0.9340278  14.2335565  -7.4455829   0.9340278
  14.2335565  -7.4455829   0.9340278  14.2335565  -7.4455829   0.9340278
  14.2335565  -7.4455829   0.9340278  14.2335565  -7.4455829   0.9340278
  14.2335565  -7.4455829   0.9340278  14.2335565  -7.4455829   0.9340278
  14.2335565  -7.4455829   0.9340278  10.8280725  -5.9760694  -0.1152739
  10.8280725  -5.9760694  -0.1152739  10.8280725  -5.9760694  -0.1152739
  10.8280725  -5.9760694  -0.1152739  10.8280725  -5.9760694  -0.1152739
  10.2887581  -6.0392196  -1.2668688  10.2887581  -6.0392196  -1.2668688
  10.2887581  -6.0392196  -1.2668688  10.2887581  -6.0392196  -1.2668688
  10.2887581  -6.0392196  -1.2668688  11.6763982  -5.0643968   0.3636369
  12.3471871  -5.4032748   1.0295682  11.3548590  -3.6051516   0.2277453
  10.4115987  -3.1792488   0.5832214  12.1572257  -3.0794666   0.9059005
  11.5689476  -3.3941179  -0.8437952
  -1.7712774   2.9191957  -1.0668196   0.4044132  -0.1818947   0.3181710
   0.5328014  -0.0631147   0.9463921  -1.7546763  -0.6193763  -1.4063463
   0.2004506   0.5001195   0.6444678   0.7742100   0.2520350  -0.3012739
   0.1603606  -0.3223624  -0.1925883   0.1603606  -0.3223624  -0.1925883
   0.1603606  -0.3223624  -0.1925883   0.1603606  -0.3223624  -0.1925883
   0.1603606  -0.3223624  -0.1925883   0.9945470  -2.8087442  -1.1377994
   0.9945470  -2.8087442  -1.1377994   0.9945470  -2.8087442  -1.1377994
   0.9945470  -2.8087442  -1.1377994   0.9945470  -2.8087442  -1.1377994
   0.1208818  -0.4183483  -1.1076615   0.1208818  -0.4183483  -1.1076615
   0.1208818  -0.4183483  -1.1076615   0.1208818  -0.4183483  -1.1076615
   0.1208818  -0.4183483  -1.1076615   2.8184837   3.8066868  -2.5212938
   2.8184837   3.8066868  -2.5212938   2.8184837   3.8066868  -2.5212938
   2.8184837   3.8066868  -2.5212938   2.8184837   3.8066868  -2.5212938
   0.1284270   0.0449300   0.8193599   0.1284270   0.0449300   0.8193599
   0.1284270   0.0449300   0.8193599   0.1284270   0.0449300   0.8193599
   0.1284270   0.0449300   0.8193599  -1.1393389   0.0916035  -2.2150225
  -1.1393389   0.0916035  -2.2150225  -1.1393389   0.0916035  -2.2150225
  -1.1393389   0.0916035  -2.2150225  -1.1393389   0.0916035  -2.2150225
  -0.0404279  -1.3065165   0.7275426  -0.0404279  -1.3065165   0.7275426
  -0.0404279  -1.3065165   0.7275426  -0.0404279  -1.3065165   0.7275426
  -0.0404279  -1.3065165   0.7275426   0.5791179  -0.3783973  -0.0621044
   0.5791179  -0.3783973  -0.0621044   0.5791179  -0.3783973  -0.0621044
   0.5791179  -0.3783973  -0.0621044   0.5791179  -0.3783973  -0.0621044
   0.5791179  -0.3783973  -0.0621044   0.5791179  -0.3783973  -0.0621044
   0.5791179  -0.3783973  -0.0621044   0.5791179  -0.3783973  -0.0621044
   0.5791179  -0.3783973  -0.0621044  -1.0799228  -4.1232246   0.3974789
  -1.0799228  -4.1232246   0.3974789  -1.0799228  -4.1232246   0.3974789
  -1.0799228  -4.1232246   0.3974789  -1.0799228  -4.1232246   0.3974789
  -1.0799228  -4.1232246   0.3974789  -1.0799228  -4.1232246   0.3974789
  -1.0799228  -4.1232246   0.3974789  -1.0799228  -4.1232246   0.3974789
  -1.0799228  -4.1232246   0.3974789   0.1543078  -0.2483131  -0.0025897
   0.1543078  -0.2483131  -0.0025897   0.1543078  -0.2483131  -0.0025897
   0.1543078  -0.2483131  -0.0025897   0.1543078  -0.2483131  -0.0025897
   0.1543078  -0.2483131  -0.0025897   0.1543078  -0.2483131  -0.0025897
   0.1543078  -0.2483131  -0.0025897   0.1543078  -0.2483131  -0.0025897
   0.1543078  -0.2483131  -0.0025897   2.1907230  -0.5285213  -0.2724395
   2.1907230  -0.5285213  -0.2724395   2.1907230  -0.5285213  -0.2724395
   2.1907230  -0.5285213  -0.2724395   2.1907230  -0.5285213  -0.2724395
   2.1907230  -0.5285213  -0.2724395   2.1907230  -0.5285213  -0.2724395
   2.1907230  -0.5285213  -0.2724395   2.1907230  -0.5285213  -0.2724395
   2.1907230  -0.5285213  -0.2724395   1.7956173  -1.6149945  -0.4435586
   1.7956173  -1.6149945  -0.4435586   1.7956173  -1.6149945  -0.4435586
   1.7956173  -1.6149945  -0.4435586   1.7956173  -1.6149945  -0.4435586
   1.7956173  -1.6149945  -0.4435586   1.7956173  -1.6149945  -0.4435586
   1.7956173  -1.6149945  -0.4435586   1.7956173  -1.6149945  -0.4435586
   1.7956173  -1.6149945  -0.4435586  -0.8493814  -0.6325898   1.3670542
  -0.8493814  -0.6325898   1.3670542  -0.8493814  -0.6325898   1.3670542
  -0.8493814  -0.6325898   1.3670542  -0.8493814  -0.6325897   1.3670542
  -0.8493814  -0.6325898   1.3670542  -0.8493814  -0.6325898   1.3670542
  -0.8493814  -0.6325898   1.3670542  -0.8493814  -0.6325898   1.3670542
  -0.8493814  -0.6325898   1.3670542   0.5267384  -0.6163459   0.2037213
   0.5267384  -0.6163459   0.2037213   0.5267384  -0.6163459   0.2037213
   0.5267384  -0.6163459   0.2037213   0.5267384  -0.6163459   0.2037213
   0.5267384  -0.6163459   0.2037213   0.5267384  -0.6163459   0.2037213
   0.5267384  -0.6163459   0.2037213   0.5267384  -0.6163459   0.2037213
   0.5267384  -0.6163459   0.2037213   0.0358815  -0.8791662   1.7219926
   0.0358815  -0.8791662   1.7219926   0.0358815  -0.8791662   1.7219926
   0.0358815  -0.8791662   1.7219926   0.0358815  -0.8791662   1.7219926
   0.0358815  -0.8791662   1.7219926   0.0358815  -0.8791662   1.7219926
   0.0358815  -0.8791662   1.7219926   0.0358815  -0.8791662   1.7219926
   0.0358815  -0.8791662   1.7219926   1.6586132  -1.1140655   0.9245807
   1.6586132  -1.1140655   0.9245807   1.6586132  -1.1140655   0.9245807
   1.6586132  -1.1140655   0.9245807   1.6586132  -1.1140655   0.9245807
   1.6586132  -1.1140655   0.9245807   1.6586132  -1.1140655   0.9245807
   1.6586132  -1.1140655   0.9245807   1.6586132  -1.1140655   0.9245807
   1.6586132  -1.1140655   0.9245807  -1.8831908  -1.2756427   0.9472177
  -1.8831908  -1.2756427   0.9472177  -1.8831908  -1.2756427   0.9472177
  -1.8831908  -1.2756427   0.9472177  -1.8831908  -1.2756427   0.9472177
  -1.8831908  -1.2756427   0.9472177  -1.8831908  -1.2756427   0.9472177
  -1.8831908  -1.2756427   0.9472177  -1.8831908  -1.2756427   0.9472177
  -1.8831908  -1.2756427   0.9472177  -0.3654523   0.3445947  -0.1108282
  -0.3654523   0.3445947  -0.1108282  -0.3654523   0.3445947  -0.1108282
  -0.3654523   0.3445947  -0.1108282  -0.3654523   0.3445947  -0.1108282
  -0.2991234   0.2577884   0.0579521  -0.2991234   0.2577884   0.0579521
  -0.2991234   0.2577884   0.0579521  -0.2991234   0.2577884   0.0579521
  -0.2991234   0.2577884   0.0579521  -0.1222184  -0.6726668   0.4806893
  -0.6360695  -1.0184750  -0.9281615  -0.3689031   0.3368593   0.6503111
   1.2802003   3.7239601   2.7575264  -0.0602628   0.2454519   3.1568154
  -0.6959169  -0.3959172  -2.5280680
