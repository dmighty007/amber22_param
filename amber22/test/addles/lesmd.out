
          -------------------------------------------------------
          Amber 9  SANDER                 Scripps/UCSF 2004
          -------------------------------------------------------

| Run on 04/07/2005 at 10:23:12
  [-O]verwriting output

File Assignments:
|  MDIN: lesmd.in                                                              
| MDOUT: lesmd.out                                                             
|INPCRD: les.equ.crd                                                           
|  PARM: les.prm                                                               
|RESTRT: lesmd.restrt                                                          
|  REFC: refc                                                                  
| MDVEL: lesmd.vel                                                             
|  MDEN: lesmd.en                                                              
| MDCRD: lesmd.crd                                                             
|MDINFO: lesmd.info                                                            
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                

 
 Here is the input file:
 
leucine LES test case                                                          
 &cntrl                                                                        
                                                                               
  ntx    = 5, ntwx   = 10,   nsnb   = 10,                dielc = 1.0,          
              nstlim = 1000, ntpr   = 10, temp0  = 298., dt    = 0.001,        
  ntt    = 1,                ntc    = 1,  tol    = 0.0005,                     
  scee   = 1.2,                                                                
                                                                               
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags: LES                                                                    
 getting new box info from bottom of inpcrd
| peek_ewald_inpcrd: Box info not found in inpcrd
|Largest sphere to fit in unit cell has radius =    -0.991
